File: treinos/treino008/app/src/main/java/aluno/gabriel/treino008/datamodel/UsuarioDatamodel.java
package aluno.gabriel.treino008.datamodel;

import aluno.gabriel.treino008.core.ApplicationUtil;

public class UsuarioDatamodel {
    public static String queryCreateTable(){
        ApplicationUtil.query += "CREATE TABLE IF NOT EXISTS "+ApplicationUtil.TABELA+" ( ";
        ApplicationUtil.query += ApplicationUtil.ID+" INTEGER PRIMARY KEY AUTOINCREMENT, ";
        ApplicationUtil.query += ApplicationUtil.NOME+" TEXT NOT NULL, ";
        ApplicationUtil.query += ApplicationUtil.EMAIL+" TEXT NOT NULL UNIQUE, ";
        ApplicationUtil.query += ApplicationUtil.SENHA+" TEXT NOT NULL ";
        ApplicationUtil.query += " ) ";
        return ApplicationUtil.query;
    }
}

File: treinos/treino010/app/src/main/java/aluno/gabriel/treino010/datamodel/UsuarioDatamodel.kt
package aluno.gabriel.treino010.datamodel

import aluno.gabriel.treino010.core.AppUtil

class UsuarioDatamodel {
    companion object {
        fun queryCreateTable(): String {
            AppUtil.query += "CREATE TABLE IF NOT EXISTS " + AppUtil.databaseTable + " ( "
            AppUtil.query += AppUtil.ID + " INTEGER PRIMARY KEY AUTOINCREMENT, "
            AppUtil.query += AppUtil.NOME + " TEXT NOT NULL, "
            AppUtil.query += AppUtil.EMAIL + " TEXT NOT NULL UNIQUE, "
            AppUtil.query += AppUtil.SENHA + " TEXT NOT NULL "
            AppUtil.query += " ) "
            return AppUtil.query
        }
    }
}